<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculator4</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowminimumheight="20,30,30,30,30,30,30">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <item row="5" column="2">
    <widget class="QPushButton" name="digitBtn9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="leftBracketBtn">
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="digitBtn8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="digitBtn0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="digitBtn2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="digitBtn1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="powBtn">
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="rightBracketBtn">
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="addBtn">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="digitBtn5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="digitBtn6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="digitBtn3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="digitBtn7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pointBtn">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="digitBtn4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="subtractionBtn">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="multiplicationBtn">
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="signBtn">
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="divisionBtn">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="equalBtn">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="clearAllBtn">
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="clearBtn">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
